pkg/server/container_create.go

```diff
@@ -161,6 +161,12 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	}
 
 	meta.ProcessLabel = spec.Process.SelinuxLabel
+
+	// handle any KVM based runtime
+	if err := modifyProcessLabel(ociRuntime.Type, spec); err != nil {
+		return nil, err
```

Comment:
is there an intent to fail here if there is a spec.Process.SelinuxLabel but selinux is not enabled? skimming over the code looks like there may be a case where ChangeToKVM could bubble up an error when we might be ok ignoring?

Reply:
let me double check

Reply:
I think this would be an error anyways but I'm updating the code to handle this case.